src/main/java/seedu/savenus/logic/parser/ParserUtil.java

```diff
@@ -47,6 +49,25 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
+     * trimmed.
+     * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
+     */
+    public static List<Index> parseIndexes(String oneBasedIndex) throws ParseException {
+        String trimmedIndexes = oneBasedIndex.trim();
+        String[] indexes = trimmedIndexes.split("\\s+");
+        List<Index> outputIndexes = new ArrayList<Index>();
+        for (int i = 0; i < indexes.length; i++) {
+            String trimmedIndex = indexes[i];
+            if (!StringUtil.isNonZeroUnsignedInteger(trimmedIndex)) {
+                throw new ParseException(MESSAGE_INVALID_INDEX);
+            }
```

Comment:
If the foodlist is empty, you should throw another catch though, cause its not that the food index is invalid, but its because the foodlist is empty! Maybe change the catch for that?

Reply:
Will fix in next commit, thank you for the catch!